pkg/util/test/util.go

```diff
@@ -521,7 +522,18 @@ func expectedPods(cluster *corev1.StorageCluster) (int, error) {
 	return podCount, nil
 }
 
-func validateComponents(cluster *corev1.StorageCluster, timeout, interval time.Duration) error {
+func getImageNameFromMap(pxImageList map[string]string, component string) string {
```

Comment:
why do we need this? can we simple use `image := pxImageList[component]`?

Reply:
if you need to validade if the key is present you can use
```
if value, ok := pxImageList[component]; ok {
         image  =  value
}
```

Reply:
Thats a good idea @thiguetta let me try this out!